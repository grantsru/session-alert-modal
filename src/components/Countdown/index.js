import React from "react";
import { formatTime } from "../../utils";
import Typography from "@material-ui/core/Typography";

const Index = ({ time, timeStringFormatter = formatTime }) => (
  <Typography style={{ margin: "15px 0 20px 0" }} variant={"subheading"} align={"center"}>
    <strong>Time Remaining:</strong> {timeStringFormatter(time)}
  </Typography>
);

export default Index;
