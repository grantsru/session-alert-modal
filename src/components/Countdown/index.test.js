import React from "react";
import Countdown from "./index";
import { create } from "react-test-renderer";

const logout = () => null;
const extend = () => null;
const handleButtonClick = () => null;
const warningText = "foo";
const timeUntilExpired = 10;

describe("Countdown", () => {
  test("it matches the snapshot", () => {
    const component = create(
      <Countdown
        time={10}
      />
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
