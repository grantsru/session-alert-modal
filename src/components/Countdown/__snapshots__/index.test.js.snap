// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Countdown it matches the snapshot 1`] = `
<h3
  className="MuiTypography-root-1 MuiTypography-subheading-8 MuiTypography-alignCenter-24"
  style={
    Object {
      "margin": "15px 0 20px 0",
    }
  }
>
  <strong>
    Time Remaining:
  </strong>
   
  10 seconds
</h3>
`;
